<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMap</class>
 <widget class="QMainWindow" name="CreateMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>591</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="create">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>530</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Britannic Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_level">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>530</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Britannic Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Choose name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>311</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="function1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/image/function1.png</normaloff>:/image/function1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>47</width>
             <height>47</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxFunction1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="function2">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/image/function2.png</normaloff>:/image/function2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>47</width>
             <height>47</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxFunction2">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="function3">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/image/function3.png</normaloff>:/image/function3.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>47</width>
             <height>47</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxFunction3">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>7</number>
         </property>
         <item>
          <widget class="QCheckBox" name="function4">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/image/function4.png</normaloff>:/image/function4.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>47</width>
             <height>47</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxFunction4">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>7</number>
         </property>
         <item>
          <widget class="QCheckBox" name="function5">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/image/function5.png</normaloff>:/image/function5.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>47</width>
             <height>47</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxFunction5">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="arrow_up">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/image/arrow_up.png</normaloff>:/image/arrow_up.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="arrow_right">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/image/arrow_right.png</normaloff>:/image/arrow_right.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="arrow_left">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/image/arrow_left.png</normaloff>:/image/arrow_left.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="blue_brush">
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/image/blue_brush.png</normaloff>:/image/blue_brush.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="dark_magenta_brush">
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/image/dark_magenta_brush.png</normaloff>:/image/dark_magenta_brush.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="red_brush">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/image/red_brush.png</normaloff>:/image/red_brush.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="dark_yellow_brush">
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/image/dark_yellow_brush.png</normaloff>:/image/dark_yellow_brush.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="blue">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>51</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
         <property name="autoRepeatInterval">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="dark_magenta">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="red">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="dark_yellow">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Britannic Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose size of map:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Britannic Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
